### PR TITLE
Fix broken links when rendered on the reg

### PR DESCRIPTION
**Pull request by ml4**
## Description
Links in the README with relative paths are causing 404s for users consuming the public registry. This PR fixes them to absolute paths.

## Related issue
Highlighted in [this issue for the terraform-aws-terraform-enterprise-hvd module](https://github.com/hashicorp/terraform-aws-terraform-enterprise-hvd/issues/8)

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [x] This change requires a documentation update

## How has this been tested?
- Vercel rendering pending
- Reconcile all link changes with Vercel

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

## Additional notes
Links still need testing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,8 +55,8 @@ Azure Key Vault containing the following TFE _bootstrap_ secrets:
  - **TFE TLS certificate** - base64-encoded string of certificate file in PEM format
  - **TFE TLS private key** - base64-encoded string of private key file in PEM format
  - **TFE custom CA bundle** - base64-encoded string of custom CA bundle file in PEM format
- 
- >📝 Note: See the [TFE TLS Certificate Rotation](./docs/tfe-cert-rotation.md) doc for instructions on how to base64-encode the certificates with proper formatting before storing them as Key Vault secrets.
+
+ >📝 Note: See the [TFE TLS Certificate Rotation](https://raw.githubusercontent.com/hashicorp/terraform-azurerm-terraform-enterprise-hvd/main/docs/tfe-cert-rotation.md) doc for instructions on how to base64-encode the certificates with proper formatting before storing them as Key Vault secrets.
```
Comment (ml4) [inline README.md:59]:
```suggestion
 >📝 Note: See the [TFE TLS Certificate Rotation](https://github.com/hashicorp/terraform-azurerm-terraform-enterprise-hvd/tree/main/docs/tfe-cert-rotation.md) doc for instructions on how to base64-encode the certificates with proper formatting before storing them as Key Vault secrets.
```

File: README.md [hunk 2]
```diff
@@ -82,10 +82,10 @@ One of the following logging destinations for the TFE container logs:
 
 1. Create/configure/validate the applicable [prerequisites](#prerequisites).
 
-2. Nested within the [examples](./examples/) directory are subdirectories containing ready-made Terraform configurations for example scenarios on how to call and deploy this module. To get started, choose the example scenario that most closely matches your requirements. You can customize your deployment later by adding additional module [inputs](#inputs) as you see fit (see the [Deployment-Customizations](./docs/deployment-customizations.md) for more details).
+2. Nested within the [examples](https://raw.githubusercontent.com/hashicorp/terraform-azurerm-terraform-enterprise-hvd/main/examples/) directory are subdirectories containing ready-made Terraform configurations for example scenarios on how to call and deploy this module. To get started, choose the example scenario that most closely matches your requirements. You can customize your deployment later by adding additional module [inputs](#inputs) as you see fit (see the [Deployment-Customizations](https://raw.githubusercontent.com/hashicorp/terraform-azurerm-terraform-enterprise-hvd/main/docs/deployment-customizations.md) for more details).
```
Comment (ml4) [inline README.md:85]:
```suggestion
2. Nested within the [examples](https://github.com/hashicorp/terraform-azurerm-terraform-enterprise-hvd/tree/main/examples/) directory are subdirectories containing ready-made Terraform configurations for example scenarios on how to call and deploy this module. To get started, choose the example scenario that most closely matches your requirements. You can customize your deployment later by adding additional module [inputs](#inputs) as you see fit (see the [Deployment-Customizations](https://github.com/hashicorp/terraform-azurerm-terraform-enterprise-hvd/tree/main/docs/deployment-customizations.md) for more details).
```

File: README.md [hunk 3]
```diff
@@ -106,7 +106,7 @@ One of the following logging destinations for the TFE container logs:
...
 
-5. Populate your own custom values into the `terraform.tfvars.example` file that was provided (in particular, the values enclosed in the `<>` characters). Then, remove the `.example` file extension such that the file is now named `terraform.tfvars`. If you would like to further customize your deployment beyond what is in your chosen example scenario, see the [deployment customizations](./docs/deployment-customizations.md) doc for more details on common customizations.
+5. Populate your own custom values into the `terraform.tfvars.example` file that was provided (in particular, the values enclosed in the `<>` characters). Then, remove the `.example` file extension such that the file is now named `terraform.tfvars`. If you would like to further customize your deployment beyond what is in your chosen example scenario, see the [deployment customizations](https://raw.githubusercontent.com/hashicorp/terraform-azurerm-terraform-enterprise-hvd/main/docs/deployment-customizations.md) doc for more details on common customizations.
```
Comment (ml4) [inline README.md:109]:
```suggestion
5. Populate your own custom values into the `terraform.tfvars.example` file that was provided (in particular, the values enclosed in the `<>` characters). Then, remove the `.example` file extension such that the file is now named `terraform.tfvars`. If you would like to further customize your deployment beyond what is in your chosen example scenario, see the [deployment customizations](https://github.com/hashicorp/terraform-azurerm-terraform-enterprise-hvd/tree/main/docs/deployment-customizations.md) doc for more details on common customizations.
```

File: README.md [hunk 4]
```diff
@@ -155,11 +155,11 @@ One of the following logging destinations for the TFE container logs:
...
- - [Deployment Customizations](./docs/deployment-customizations.md)
- - [TFE Version Upgrades](./docs/tfe-version-upgrades.md)
- - [TFE TLS Certificate Rotation](./docs/tfe-cert-rotation.md)
- - [TFE Configuration Settings](./docs/tfe-config-settings.md)
- - [Azure GovCloud Deployment](./docs/govcloud-deployment.md)
+ - [Deployment Customizations](https://raw.githubusercontent.com/hashicorp/terraform-azurerm-terraform-enterprise-hvd/main/docs/deployment-customizations.md)
+ - [TFE Version Upgrades](https://raw.githubusercontent.com/hashicorp/terraform-azurerm-terraform-enterprise-hvd/main/docs/tfe-version-upgrades.md)
+ - [TFE TLS Certificate Rotation](https://raw.githubusercontent.com/hashicorp/terraform-azurerm-terraform-enterprise-hvd/main/docs/tfe-cert-rotation.md)
+ - [TFE Configuration Settings](https://raw.githubusercontent.com/hashicorp/terraform-azurerm-terraform-enterprise-hvd/main/docs/tfe-config-settings.md)
+ - [Azure GovCloud Deployment](https://raw.githubusercontent.com/hashicorp/terraform-azurerm-terraform-enterprise-hvd/main/docs/govcloud-deployment.md)
```
Comment (ml4) [inline README.md:162]:
```suggestion
 - [Deployment Customizations](https://github.com/hashicorp/terraform-azurerm-terraform-enterprise-hvd/tree/main/docs/deployment-customizations.md)
 - [TFE Version Upgrades](https://github.com/hashicorp/terraform-azurerm-terraform-enterprise-hvd/tree/main/docs/tfe-version-upgrades.md)
 - [TFE TLS Certificate Rotation](https://github.com/hashicorp/terraform-azurerm-terraform-enterprise-hvd/tree/main/docs/tfe-cert-rotation.md)
 - [TFE Configuration Settings](https://github.com/hashicorp/terraform-azurerm-terraform-enterprise-hvd/tree/main/docs/tfe-config-settings.md)
 - [Azure GovCloud Deployment](https://github.com/hashicorp/terraform-azurerm-terraform-enterprise-hvd/tree/main/docs/govcloud-deployment.md)
```

**Review by jeffmccollum — APPROVED** [review]

**Comment by roooms** [comment]
Closing this and approaching the relative links issue differently. I'll make the support commits on another PR as well.

**Pull request closed without merging**